### ISSUE TITLE
launch apps on MacOS

### ISSUE THREAD

**Issue by jeremyfix** [jeremyfix/dcejs#7]
On MacOS, for starting the app : 

- TigerVNC is installed into : /Applications/TigerVNC Viewer 1.11.0.app/Contents/MacOS/TigerVNC\ Viewer
- NoMachine /Applications/NoMachine.app/Contents/MacOS/nxplayer

**Comment by jeremyfix** [comment]
The installed apps can be requested with  : `system_profiler SPApplicationsDataType -xml` which gives the output below and can be parsed with [jsonpath](https://stackoverflow.com/questions/44719196/using-electron-node-js-how-can-i-detect-all-installed-browsers-on-macos)

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<array>
	<dict>
		<key>_SPCommandLineArguments</key>
		<array>
			<string>/usr/sbin/system_profiler</string>
			<string>-nospawn</string>
			<string>-xml</string>
			<string>SPApplicationsDataType</string>
			<string>-detailLevel</string>
			<string>full</string>
		</array>
		<key>_SPCompletionInterval</key>
		<real>4.5778800249099731</real>
		<key>_SPResponseTime</key>
		<real>4.7002439498901367</real>
		<key>_dataType</key>
		<string>SPApplicationsDataType</string>
		<key>_detailLevel</key>
		<integer>1</integer>
		<key>_items</key>
		<array>
			<dict>
				<key>_name</key>
				<string>Préférences Système</string>
				<key>has64BitIntelCode</key>
				<string>yes</string>
				<key>lastModified</key>
				<date>2020-02-04T08:21:24Z</date>
				<key>obtained_from</key>
				<string>apple</string>
				<key>path</key>
				<string>/Applications/System Preferences.app</string>
				<key>runtime_environment</key>
				<string>arch_x86</string>
				<key>signed_by</key>
				<array>
					<string>Software Signing</string>
					<string>Apple Code Signing Certification Authority</string>
					<string>Apple Root CA</string>
				</array>
				<key>version</key>
				<string>14.0</string>
			</dict>
			<dict>
				<key>_name</key>
				<string>NoMachine</string>
				<key>has64BitIntelCode</key>
				<string>yes</string>
				<key>info</key>
				<string>NoMachine 7.6.2, Copyright (c) 2002-2021 NoMachine</string>
				<key>lastModified</key>
				<date>2021-08-30T14:03:05Z</date>
				<key>obtained_from</key>
				<string>identified_developer</string>
				<key>path</key>
				<string>/Applications/NoMachine.app</string>
				<key>runtime_environment</key>
				<string>arch_x86</string>
				<key>signed_by</key>
				<array>
					<string>Developer ID Application: NoMachine S.a.r.l. (493C5JZAGR)</string>
					<string>Developer ID Certification Authority</string>
					<string>Apple Root CA</string>
				</array>
				<key>version</key>
				<string>7.6.2</string>
			</dict>
<dict>
				<key>_name</key>
				<string>TigerVNC Viewer 1.11</string>
				<key>has64BitIntelCode</key>
				<string>yes</string>
				<key>info</key>
				<string>1.11.0, Copyright © 1998-2020 [many holders]</string>
				<key>lastModified</key>
				<date>2020-09-08T23:42:00Z</date>
				<key>obtained_from</key>
				<string>identified_developer</string>
				<key>path</key>
				<string>/Applications/TigerVNC Viewer 1.11.0.app</string>
				<key>runtime_environment</key>
				<string>arch_x86</string>
				<key>signed_by</key>
				<array>
					<string>Developer ID Application: Brian Hinz (S5LX88A9BW)</string>
					<string>Developer ID Certification Authority</string>
					<string>Apple Root CA</string>
				</array>
				<key>version</key>
				<string>1.11.0</string>
			</dict>
		</array>
		<key>_timeStamp</key>
		<date>2021-08-30T14:30:12Z</date>
		<key>_versionInfo</key>
		<dict>
			<key>com.apple.SystemProfiler.SPApplicationsReporter</key>
			<string>915</string>
		</dict>
	</dict>
</array>
</plist>
```

**Comment by jeremyfix** [comment]
Should be fixed in 9cadd0078eb383d3fb917a5a420d578fa58f681c

**Comment by jeremyfix** [comment]
fixed.
